pages/index.tsx

```diff
@@ -19,63 +19,65 @@ function	VaultCard({currentVault}: {currentVault: TVault}): ReactElement {
 
 	return (
 		<div className={'w-full'}>
-			<Link href={`/vault/${currentVault.address}`}>
-				<Card className={'yearn--card col-span-1 md:col-span-3'} padding={'none'}>
-					<motion.div initial={'rest'} whileHover={'hover'} animate={'rest'} className={'cursor-pointer'}>
-						<motion.div
-							variants={slashMotion}
-							className={'macarena--vaultCard flex flex-col items-start justify-between p-6 pb-4'}>
-							<div className={'flex w-full flex-row items-start justify-between'}>
-								<div className={'min-h-[32px] min-w-[32px] md:min-h-[80px] md:min-w-[80px]'}>
-									<Image
-										src={currentVault.token.icon}
-										width={80}
-										height={80} />
+			<Link href={`/vault/${currentVault.address}`} passHref>
+				<a>
```

Comment:
Looks like a lot has changed, but literally was just wrapping this in an `a` tag :)